assets/js/partials/mode-switcher.js

```diff
@@ -0,0 +1,56 @@
+//@preserve Mode Switcher by derekkedziora
```

Comment:
If we're using an external library, maybe we should include the `min.js` version instead with a link to the repository in the readme.

Reply:
More an example than an external library.
https://github.com/derekkedziora/jekyll-demo/blob/master/scripts/mode-switcher.js

I modified it to work with the FontAwesome button.